index.html

```diff
@@ -30,16 +42,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="teal">Box 1</div>
+                <div class="gold">Box 2</div>
+                <div class="cadetblue">Box 3</div>
+                <div class="coral">Box 4</div>
+                <div class="crimson">Box 5</div>
+                <div class="forestgreen">Box 6</div>
+                <div class="darkorchid">Box 7</div>
+                <div class="hotpink">Box 8</div>
+                <div class="indigo">Box 9</div>
+                <div class="dodgerblue">Box 10</div>
```

Comment:
Here to could have give all the boxes a `box` class for all the shared styling puls the color class to add each color to each specific box